libs/embed/src/embed.ts

```diff
@@ -45,6 +47,7 @@ class Novu {
 
   init = (
     clientId: string,
+    backendUrl: string,
```

Comment:
So @AlexVCS what I was thinking that instead of adding a new parameter we can pass the backendUrl as optional parameter inside IOptions. For example:

```
novu.init('<REPLACE_APPLICATION_ID>', {
    unseenBadgeSelector: '#unseen-badge',
    bellSelector: '#notification-bell',
    backednUrl: 'http://custom.com', <--------- THIS
    socketUrl:  'http://custom.com', <--------- THIS
    position: {
      top: '50px',
      left: '100px'
    }
  }, {
    ...subscriberProps
  });
```

Reply:
Ok so you're saying to add those two lines into docs/docs/notification-center/iframe-embed.md? And then I won't need line 50 in libs/embed/src/embed.ts. Think I understand!